pgcontents/tests/test_synchronization.py

```diff
@@ -214,6 +214,13 @@ def encrypted_pgmanager(user_id):
         paths = [(user_id, path)
                  for user_id in user_ids
                  for path in populate(managers[user_id])]
+
+        # Create a text file for each user as well, which should be ignored by
+        # the notebook generators
+        model = {'content': 'text file contents', 'format': 'text'}
+        for manager in managers.values():
+            manager.new(model, path='text file.txt')
```

Comment:
Oh I see, this won't even actually hit that exception anymore due to the `if table is files` check. That makes this feel like a bit of a passive test, meaning if no exception was raised that means it worked. That is, I verified the exception is hit when the `if table is files` check is removed, but I don't see a great way to make that explicit in the tests.